vyper_fetch.sh

```diff
@@ -1,10 +1,16 @@
 #!/bin/bash
 
+if [ $# -eq 0 ]
+  then
+    echo "Expected one argument: output file"
+    exit 1
+fi
+
 body=$(curl --silent --user-agent "blockscout-solc-bin" "https://api.github.com/repos/vyperlang/vyper/releases" |
```

Comment:
does it download all the compilers on run? even when there is no updates? can we make it so it would download only new versions?

Reply:
using cache can help, but still, cache can be invalidated and then it will be bad

Reply:
> does it download all the compilers on run? 

Only the first time, then using cache and wget `--no-clobber` option there will be skips (based on names)

As per [documentation](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy) there is no mention of a sudden cache invalidation.
But if we want, I can add some smarter logic